rsocket-core/src/main/java/io/rsocket/loadbalance/LoadbalanceRSocketClient.java

```diff
@@ -179,4 +186,12 @@ private LoadbalanceStrategy initLoadbalanceStrategy() {
           : new RoundRobinLoadbalanceStrategy());
     }
   }
+
+  /**
+   * Extension for {@link LoadbalanceStrategy} which allows pre-setup {@link RSocketConnector} for
+   * {@link LoadbalanceStrategy} needs
+   */
+  public interface InitializingLoadbalanceStrategy extends LoadbalanceStrategy {
+    void init(RSocketConnector connector);
+  }
```

Comment:
Let's make this a top-level interface. It has become a bit long otherwise. `ClientLoadbalanceStrategy` as we discussed previously is a good name because it's for use from the client side.

Reply:
fixed